### DISCUSSION TITLE
Toolkit

### DISCUSSION THREAD

**Discussion by peyman** [GRC-Engineer/awesome-security-GRC#14] [Q&A] [answered]
Hi
Wondering if there is a set of toolkits that would be recommended?
Such tools would help to ease the effort involved with this stream of work. Could be for tracking, reporting, efficiency gain, etc.
Thanks heaps.

**Comment by ryanmrestivo** [comment, marked as answer]
Hey Peyman!  There are a lot of options that work well but it is important to consider the size/maturity/goals of the orginization before deciding on a solution.  If you want to specifcy a little more maybe we can help better :)
Broadly speaking - my 2 cents if you want to look into more: (I don't necessarily endorse any of these, but have some familiarity with those below!)

Microsoft has a free account for looking at cloud policies (more technical compliance)
https://azure.microsoft.com/en-ca/services/azure-policy/#overview

If your orginzation has Azure - Microsoft Azure DevOps platform for managing documentation and working with large teams...
https://azure.microsoft.com/en-us/services/devops/

lots of reviews from a trusted advisor:
https://www.gartner.com/reviews/market/integrated-risk-management

Some larger GRC platforms -

RSA Archer:
https://www.archerirm.com/
IT and security risk management 
Enterprise and operational  risk management 
Third-party governance  
Audit management  
Compliance management  
Business resiliency  
Public sector risk management

IBM OpenPages: (free trial)
https://www.ibm.com/products/openpages-with-watson

**Reply by peyman** [reply]
That's a very good start. Thanks Ryan. 👍
